pytest-itde/README.rst

```diff
@@ -1,7 +1,7 @@
-pytest-itde Plugin
-==================
+pytest-exasol-itde Plugin
```

Comment:
This heading does not seem to be consistent with the one in saas:

`pytest-exasol-itde Plugin`
vs
`pytest-saas Plugin`

Reply:
Thanks. See next push with updated file `pytest-saas/README.md` consistently using `pytest-exasol-saas`.